conf/default.yml

```diff
@@ -6,11 +6,19 @@ app.contact.dns.socket: 0.0.0.0:8853
 app.contact.gist: API_KEY
 app.contact.html: /weather
 app.contact.http: http://0.0.0.0:8888
+app.contact.slack.api_key: SLACK_TOKEN
```

Comment:
Did you rebase to master? That should get rid of these extra things

Reply:
This is the config in master, I had to because there were merge conflicts